loader/loader.go

```diff
@@ -1254,14 +1254,14 @@ func (q *jobQueue) startConsumers(handler func(ctx context.Context, job *restore
 					// test condition for `job.session` means db session still could be controlled outside,
 					// it's used in unit test for now.
 					if session == nil && job.session == nil {
-						baseConn, err := job.loader.toDB.GetBaseConn(q.ctx)
-						if err != nil {
-							return err
+						baseConn, err2 := job.loader.toDB.GetBaseConn(q.ctx)
+						if err2 != nil {
```

Comment:
`err` and `err2` may reduce a little readability. I think we'd better remove the unnecessary pre-declared err and return err explicitly to avoid accidental variable shadowing.

Reply:
address in 0670fb4